### PR TITLE
review

### PR DESCRIPTION
**Pull request by GigiPoon**
code

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const Form = (props) => {
                 value={userInput}
             />
             <button>Add</button>
-            {props.errorState ? <p>no data found</p> : null}
+            {props.errorState ? <p className='errP'>no data found</p> : null}
```
Comment (priscilamattos) [inline src/components/Form.js:32]: To improve your code, you can remove props and use only the errorState. Example:
`const Form = (props) => `
to
`const Form = ({errorState, setUserSubmit})`

**Review by priscilamattos — APPROVED** [review]
Approved with comments.

**Review by priscilamattos — APPROVED** [review]
Approved with comments

**Comment by GigiPoon** [comment]
reopening

**Pull request closed without merging**